runtime/src/bloom.rs

```diff
@@ -16,7 +16,8 @@ pub trait BloomHashIndex {
 #[derive(Serialize, Deserialize, Default, Clone, Debug, PartialEq)]
 pub struct Bloom<T: BloomHashIndex> {
     pub keys: Vec<u64>,
-    pub bits: BitVec<u8>,
+    pub bits: BitVec<u64>,
+    num_bits_set: u64,
```

Comment:
Do we use this anywhere?

Reply:
its being used in crds gossip push and pull